control/action.yaml

```diff
@@ -0,0 +1,44 @@
+openapi: 3.1.0
+
+info:
+  title: Action models
+  description: >-
+    All Detail Action schemas
+  version: ^0.0.0
+
+components:
+  schemas:
+    Lacp.Action.Request:
```

Comment:
this is lacp specific and should be renamed as device state at the top level. choices can then reflect specific states.

Reply:
done.